src/ReinforcementLearningCore/src/core/run.jl

```diff
@@ -114,6 +114,7 @@ function _run(policy::AbstractPolicy,
 
         push!(policy, PostEpisodeStage(), env)  # let the policy see the last observation
         push!(hook, PostEpisodeStage(), policy, env)
+        optimise!(policy, PostActStage())
```

Comment:
This should be
```suggestion
        optimise!(policy, PostEpisodeStage())
```
right?